ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yml

```diff
@@ -99,43 +100,22 @@ spec:
           DPTP_VER="4.9"
           CP_VER="$(params.ocp_version)"
 
-          if [ "$(printf '%s\n' "$DPTP_VER" "$CP_VER" | sort -V | head -n1)" = "$DPTP_VER" ]; then
+          if [ "$(printf '%s\n' "$DPTP_VER" "$CP_VER" | sort -V | head -n1)" == "$DPTP_VER" ]; then
               OCP_VER="${DPTP_VER}"
           else
               OCP_VER="${CP_VER}"
           fi
 
-          if [ "$(params.ocp_version)" == "4.6" ] || [ "$(params.ocp_version)" == "4.7" ]; then
-              JOB_SUFFIX="aws"
-          else
-              JOB_SUFFIX="claim"
-          fi
-
           set -e
 
           if [ -f "$(workspaces.credentials.path)/project_dockerconfigjson.gpg.b64" ]
           then
               PFLT_DOCKERCONFIG="$(cat "$(workspaces.credentials.path)/project_dockerconfigjson.gpg.b64")"
               export PFLT_DOCKERCONFIG
 
-              preflight-trigger \
-          --environment "$(params.preflight_trigger_environment)" \
-          --ocp-version "${OCP_VER}" \
-          --pflt-docker-config "${PFLT_DOCKERCONFIG}" \
-          --pflt-log-level "$(params.log_level)" \
-          --pflt-index-image "$(params.bundle_index_image)" \
-          --test-asset "$(params.bundle_image)" \
-          --asset-type "$(params.asset_type)" \
-          --output-path prowjob-base-url
+              preflight-trigger create job --environment "$(params.preflight_trigger_environment)" --ocp-version "${OCP_VER}" --pflt-log-level "$(params.log_level)" --pflt-index-image "$(params.bundle_index_image)" --test-asset "$(params.bundle_image)" --asset-type "$(params.asset_type)" --output-path prowjob-base-url
```

Comment:
Is it possible to break the line after each option? It is much easier to read and track code changes.